src/pages/Profile.tsx

```diff
@@ -184,25 +189,9 @@ const Profile = (): ReactElement => {
                   flexDirection="column"
                   placeholder="Email"
                   type="email"
-                  {...register('email')}
+                  {...register('email', { pattern: /^\S+@\S+\.\S+$/ })}
```

Comment:
You can remove this regex and use YUP resolver

Reply:
merde je l'ai laissé